views/project/_form.php

```diff
@@ -77,7 +77,7 @@ function updateURI(){
             'onkeyup' => 'changeSpaces(this.value, this.id);',
             'onChange' => 'updateURI()', 
             'data-toogle' => 'tooltip', 
-            'title' => 'No spaces allowed. Used for the URI, example : drops', 
+            'title' => Yii::t('message','No spaces allowed. Used for the URI. Example : drops'), 
```

Comment:
change message to **app/messages**

Reply:
corrected in a2bbc71dd6a98681b7a1aeafafc6a157e673e628